tests/tests/client-struct-unmount-race.sh

```diff
@@ -0,0 +1,66 @@
+#
+# Unmount Server and unmount a client as it's replaying to a remaining server
+#
+
+t_require_mounts 3
```

Comment:
I'd just get rid of this t_require mounts, the node count tests that skip on fail catch all the cases and add the messages to say what's needed.   It's not just node count, it's the mix of nodes.